src/infection-model/seir.tsx

```diff
@@ -307,31 +308,34 @@ export function getAllBracketCurves(inputs: CurveProjectionInputs) {
   ).forEach(([pop, cases, recovered, deaths], index) => {
     // distribute cases across compartments proportionally
 
-    const infectious = cases * pInitiallyInfectious;
+    const activeCases = cases - recovered - deaths;
+
+    const infectious = activeCases * pInitiallyInfectious;
     // exposed is related to the number of infectious
     // but it can't be more than the total uninfected population
     const exposed = Math.min(infectious * ratioExposedToInfected, pop - cases);
 
     singleDayState.set(index, seirIndex.susceptible, pop - cases - exposed);
     singleDayState.set(index, seirIndex.exposed, exposed);
     singleDayState.set(index, seirIndex.infectious, infectious);
-    singleDayState.set(index, seirIndex.mild, cases * pInitiallyMild);
-    singleDayState.set(index, seirIndex.severe, cases * pInitiallySevere);
+    singleDayState.set(index, seirIndex.mild, activeCases * pInitiallyMild);
+    singleDayState.set(index, seirIndex.severe, activeCases * pInitiallySevere);
     singleDayState.set(
       index,
       seirIndex.hospitalized,
-      cases * pInitiallyHospitalized,
+      activeCases * pInitiallyHospitalized,
     );
     singleDayState.set(
       index,
       seirIndex.mildRecovered,
-      cases * pInitiallyMildRecovered,
+      recovered * pInitiallyMildRecovered,
     );
     singleDayState.set(
       index,
       seirIndex.severeRecovered,
-      cases * pInitiallySevereRecovered,
+      recovered * pInitiallySevereRecovered,
     );
+    singleDayState.set(index, seirIndex.fatalities, deaths);
```

Comment:
should there be a similar condition for deaths as for recovered above? (i.e. previously we were assigning a proportion of cases to be fatalities, now we're not doing that anymore so they get artificially zeroed out when we release this feature)